tests/platform_tests/sfp/test_sfputil.py

```diff
@@ -51,7 +51,13 @@ def test_check_sfputil_presence(duthosts, enum_rand_one_per_hwsku_frontend_hostn
     portmap, dev_conn = get_dev_conn(duthost, conn_graph_facts, enum_frontend_asic_index)
 
     logging.info("Check output of '{}'".format(cmd_sfp_presence))
-    sfp_presence = duthost.command(cmd_sfp_presence)
+    sfp_presence = duthost.command(cmd_sfp_presence, module_ignore_errors=True)
+
+    # For vs testbed, we will get expected Error code `ERROR_CHASSIS_LOAD = 2` here.
+    if duthost.facts["asic_type"] == "vs" and sfp_presence['rc'] == 2:
```

Comment:
I remember you had another PR for platform test, and it's return code were the same with this one, would it be better to add a setup check it in KVM? So we could find this error earlier and won't have so much duplicate code in platform test

Reply:
We can not get such error earlier unless we execute the command.